engines/librpma_fio.c

```diff
@@ -934,6 +941,19 @@ int librpma_fio_server_open_file(struct thread_data *td, struct fio_file *f,
 		goto err_free;
 	}
 
+	if (!is_dram && f->filetype == FIO_TYPE_FILE &&
+		(ret = rpma_mr_advise(mr, 0, mem_size,
```

Comment:
Can we split this one out? Assignment in the if statement is a bit fragile.

Reply:
Is it OK now?